CodeChallenge/Challenges/Clock/Clock.swift

```diff
@@ -36,7 +36,8 @@ struct ClockChallenge: CodeChallengeType {
         ethanSchatzlineClockEntry,
         brandonShegaClockEntry,
         matthijsClockEntry,
-        felixdumitClockEntry
+        felixdumitClockEntry,
+        juliand665ClockEntry,
```

Comment:
I didn't think Swift supported trailing commas in array literals, but I see this builds and runs so 👍